docs/en/sql-reference/00-sql-reference/10-data-types/45-data-type-geospatial.md

```diff
@@ -0,0 +1,140 @@
+
+## Supported Output Formats
+
+Databend supports three geospatial output formats—[WKT (Well-Known Text)](https://www.ogc.org/standards/sfa), [WKB (Well-Known Binary)](https://www.ogc.org/standards/sfa), and [GeoJSON](https://geojson.org/)—enabling compatibility with a wide range of geospatial applications and tools.
+
+| Object Type        | WKT Example                                                                                         | GeoJSON Example                                                                                                   |
+|--------------------|-----------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| Point              | `POINT(10 10)`                                                                                      | `{"coordinates":[10,10],"type":"Point"}`                                                                          |
+| LineString         | `LINESTRING(10 10, 20 30)`                                                                          | `{"coordinates":[[10,10],[20,30]],"type":"LineString"}`                                                           |
+| Polygon            | `POLYGON((10 10, 15 16, 22 10, 30 32))`                                                             | `{"coordinates":[[[10,10],[15,16],[22,10],[30,32],[10,10]]],"type":"Polygon"}`                                    |
+| MultiPoint         | `MULTIPOINT((10 20), (30 40), (50 60))`                                                             | `{"coordinates":[[10,20],[30,40],[50,60]],"type":"MultiPoint"}`                                                   |
+| MultiLineString    | `MULTILINESTRING((10 20, 30 40), (50 60, 70 80))`                                                   | `{"coordinates":[[[10,20],[30,40]],[[50,60],[70,80]]],"type":"MultiLineString"}`                                  |
+| MultiPolygon       | `MULTIPOLYGON(((10 20, 30 40, 50 60, 10 20)), ((15 25, 25 35, 35 45, 15 25)))`                      | `{"coordinates":[[[[10,20],[30,40],[50,60],[10,20]]],[[[15,25],[25,35],[35,45],[15,25]]]],"type":"MultiPolygon"}` |
+| GeometryCollection | `GEOMETRYCOLLECTION(POINT(10 20), LINESTRING(10 20, 30 40), POLYGON((10 20, 30 40, 50 60, 10 20)))` | `{"coordinates":[[[10,20],[30,40],[50,60],[10,20]]],"type":"Polygon"}`                                            |
+
+To switch the geospatial output format in Databend, configure the `SET geometry_output_format` setting with your desired format: `wkt`, `wkb`, or `geojson`. For example, 
```

Comment:
output format include `wkt`, `ewkt`, `wkb`, `ewkb`, `geojson`, the difference is `ewkt` and `ewkb` include a SRID, for example `SRID=4326;POINT(-44.3 60.1)`